core/deployment/banned-signatures.txt

```diff
@@ -1,6 +1,6 @@
-@defaultMessage Don't use Maven classes. They won't be available when using Gradle.
-org.apache.maven.**
-org.codehaus.plexus.**
+#@defaultMessage Don't use Maven classes. They won't be available when using Gradle.
+#org.apache.maven.**
+#org.codehaus.plexus.**
```

Comment:
Is it actually needed? I don't see any of these packages added to `core/deployment`? But I don't remember if forbidden-apis also scans the classpath.